index.bs

```diff
@@ -585,22 +585,29 @@ dictionary MLOperandDescriptor {
 </script>
 
 ## MLOperand ## {#api-mloperand}
+
+The {{MLOperand}} interface represents data that flows within the computational graph. See [[#programming-model]].
+
 <script type=idl>
 [SecureContext, Exposed=(Window, DedicatedWorker)]
 interface MLOperand {};
 </script>
 
 ## MLOperator ## {#api-mloperator}
 
-The {{MLOperator}} interface defines a type of operation such as the various types of activation function used to create other operations such as [[#api-mlgraphbuilder-conv2d]] or [[#api-mlgraphbuilder-batchnorm]].
+Objects implementing the {{MLOperator}} interface represent activation function types.
```

Comment:
I think we might want to leave some room for the future here and not prematurely limiting the notion of an `MLOperator`, which is a generic notion, to just the activation functions. It could also be confusing when the notion of the activation function appears twice in the API in different context, one concretely represented by `MLOperator` and another as standalone operations off the `MLGraphBuilder`. 

The idea behind `MLOperator` is to allow deferred definition of activation functions used during the construction of a fused operation e.g. GRU, etc. while allowing for future extension without changing the fused operation's API signature.

Reply:
Mentioning the extensibility for future-proofing is a good idea, PR updated.

Would it make any sense to specialize `MLActivation : MLOperator`?